docs/genai/deployment/deploy-serving-graph-as-job.ipynb

```diff
@@ -0,0 +1,78 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a2accff8",
+   "metadata": {},
+   "source": [
+    "# Run a serving graph as a batch job\n",
+    "You can convert a ServingRuntime to a KubejobRuntime, so that the graph can be run as a standalone job. This is a simpler implementation that can be run on th fly, and runs much faster than the same function as a serving runtime. YOu can use all job parameters that are used in serving graphs. Any resources assigned to the serving function are assigned to the new job. The original serving function remains unchanged and can still be invoked after running the job. \n",
+    "You can run the resulting Kubejob function from the UI after `to_job()` was performed.\n",
+    "\n",
+    "To run an existing graph as a job, just replace: \n",
+    "`function.deploy()` \n",
+    "qith \n",
+    "`function.to_job()`\n",
+    "\n",
+    "{py:meth}~`mlrun.runtimes.ServingRuntime.to_job` has an optional parameter `func_name` to rename the job function. If you do not specify a name, MLRun automatically appends the suffix '-batch' to the serving function name to differentiate between the functions.\n",
+    "\n",
+    "\n",
+    "Limitations:\n",
+    "- Scheduling of this function is not supported. \n",
+    "- Serving graph as a job with child functions is not supported.\n",
+    "\n",
+    "\n",
+    "Example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4b63b2a6",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import mlrun.serving.states\n",
+    "\n",
+    "project = mlrun.get_or_create_project(\n",
+    "    \"serving-graph\", \"./\", user_project=True, allow_cross_project=True\n",
+    ")   \n",
+    "    serving_func_obj = self.project.set_function(\n",
+    "    func=str(self.assets_path / \"function_with_model.py\"),\n",
+    "    name=\"srv_fn\",\n",
+    "    kind=\"serving\",\n",
+    "        image=self.image,\n",
+    "    )\n",
+    "    mode_runner_obj = mlrun.serving.states.ModelRunnerStep(\n",
+    "         name=\"model_runner_step_name\"\n",
+    "    )\n",
+    "    mode_runner_obj.add_model(\n",
+    "        endpoint_name=\"my-endpoint\",\n",
+    "        model_class=\"DummyModel\",\n",
+    "        execution_mechanism=execution_mechanism,\n",
+    "    )\n",
+    "    graph_obj = serving_func_obj.set_topology(\"flow\", engine=\"async\")\n",
+    "    graph_obj.to(mode_runner_obj).respond()\n",
+    "    job = serving_func_obj.to_job()\n",
+    "    local_input_path = str(self.assets_path / \"in.csv\")\n",
+    "\n",
+    "    inputs = {\"data\": input_path}\n",
+    "    run_object = self.project.run_function(job, inputs=inputs, local=local)\n",
+    "     assert run_object.status.results == {\n",
```

Comment:
the way to get the result have changes.
please see https://github.com/mlrun/mlrun/blob/development/docs/tutorials/06-batch-infer.ipynb
@danielperezz can help you on this notebook if needed